app/src/features/projects/view/components/Partnerships.test.tsx

```diff
@@ -2,28 +2,29 @@ import { render, waitFor, fireEvent, cleanup } from '@testing-library/react';
 import React from 'react';
 import { getProjectForViewResponse } from 'test-helpers/project-helpers';
 import Partnerships from './Partnerships';
-import { createMemoryHistory } from 'history';
-import { Router } from 'react-router-dom';
 import { codes } from 'test-helpers/code-helpers';
 import { useBiohubApi } from 'hooks/useBioHubApi';
-
-const history = createMemoryHistory();
+import { UPDATE_GET_ENTITIES } from 'interfaces/useProjectApi.interface';
 
 jest.mock('../../../../hooks/useBioHubApi');
 const mockUseBiohubApi = {
   project: {
-    getProjectForUpdate: jest.fn<Promise<object>, []>()
+    getProjectForUpdate: jest.fn<Promise<object>, []>(),
+    updateProject: jest.fn()
```

Comment:
what is the updateProject function for?  I haven't seen it used in other tests, I don't think.

Reply:
its for the actual edit

Reply:
it is in `species.test.tsx`, `projectcoordinator.test.tsx`, `iucnclassification.test.tsx` etc

Reply:
ah ok